test/models/asyncapi_test.js

```diff
@@ -73,6 +73,17 @@ describe('AsyncAPIDocument', () => {
     });
   });
 
+  describe('#hasVariable()', function () {
+    it('should return a boolean indicating if the AsyncAPI document has server variables', () => {
+      const doc = { servers: { test1: { url: 'test1', variables: { port: { desc: 'test1' } } } } };
+      const docNoServerVariables = { servers: { test: { url: 'test' } } };
+      const d = new AsyncAPIDocument(doc);
+      const d2 = new AsyncAPIDocument(docNoServerVariables);
+      expect(d.server('test1').hasVariables()).to.equal(true);
+      expect(d2.server('test').hasVariables()).to.equal(false);
+    });
+  });
```

Comment:
I think this test should be placed here: https://github.com/asyncapi/parser-js/blob/master/test/models/server_test.js

Reply:
damn, right 🤦‍♂ not my day